development/dod.adoc

```diff
@@ -0,0 +1,18 @@
+= Definition of Done and Ambient Requirements
+
+The Definition of Done (DoD) describes the criteria for tasks to be fulfilled in 
+order to be considered done and closed. All of the following criterias must be met:
+
+* Acceptance criterias from the user story are fully implemented and met.
+* Automated tests for the implemented features are in existence, are following the acceptance criterias in full and are running successfully. 
+* Code coverage is not decreased by the implementation and the implementation of the tests.
+* The code is properly documented with source code comments.
```

Comment:
Yes, the code comments thing is more about that we should be all aware that it makes sense to comment complex code at least to give hints what it is actually doing. Personal experience: I do not generally comment code, but on locations that require complex flows, I casually add code comments to explain what it is supposed to do. That makes debugging/changing a lot easier (even for myself). 
That is not something that can be measured or enforced, the goal is simply to make aware that it is helpful to do it. Maybe this needs more explaination.. @bartoszmajsak @ldimaggi